src/editor/layers_widget.cpp

```diff
@@ -311,37 +322,13 @@ EditorLayersWidget::refresh()
   m_selected_tilemap = nullptr;
   m_layer_icons.clear();
 
-  bool tsel = false;
-  TileMap* first_tm = nullptr;
   for (auto& i : m_editor.get_sector()->get_objects())
```

Comment:
Can this be marked `const auto&` ? and can the variable name be changed to `obj`?